### ISSUE TITLE
Revise Light Wallet section

### ISSUE THREAD

**Issue by CryptoGrampy** [monero-project/monero-site#2162]
I'm creating this issue to get some discussion and input on significant revisions to the light wallet section on the getmonero downloads page.

1. Remove any wallets that don't support self-hosted light wallet servers.  The higher the % of Monero users using hosted light wallet servers, the worse privacy becomes for ALL users of Monero.  Any wallet that doesn't safely support the use of a self-hosted Light Wallet Server (any server that follows the Light Wallet API spec) should not be recommended. 
 
2. Add more clarity to the privacy dangers of using the default 'hosted' light wallet server in these wallets.  It's not just that your privacy can be 'lessened'.  It's that all of your past, current and future incoming transactions tied to that view key will be recorded and saved, even if you decide to stop using the wallet.  Usage of the default behavior of that wallet lowers the privacy of all users of Monero, and additionally, significant information around a spend (not the spend amount itself) is leaked when using Light Wallets with a server out of your control.

**Comment by SamsungGalaxyPlayer** [comment]
1 would include the official wallets, Feather, Cake, Monerujo, etc. I don't see a reason to remove wallets that don't have this feature at this time. Perhaps a year after formal official wallet support (?). Unless you mean the removal of light wallets that rely on a set centralized server.

**Comment by lalanza808** [comment]
+1 - would love to see a stronger push for self-hosted LWS setups.

@SamsungGalaxyPlayer just the light wallets section:
![image](https://user-images.githubusercontent.com/4086248/236289626-2425a6b7-5893-40cb-814c-2eb8f3e97b5f.png)

**Comment by erciccione** [comment]
> Remove any wallets that don't support self-hosted light wallet servers

I disagree with the complete removal, but i agree we should expand the warning or put these wallets after the self-hostable ones.

> Add more clarity to the privacy dangers of using the default 'hosted' light wallet server in these wallets

I agree. Any suggestion about the specific phrasing? I would keep it short, or, if we want it longer, we might exand the "light wallets" Moneropedia entry to include all the information we want.

**Comment by CryptoGrampy** [comment]
Regarding complete removal:

1. If a light wallet doesn't allow users to input a custom server URL, they are essentially profiting off of the reduction of system privacy in Monero for the reasons stated above.  Why on Earth would we put thousands of XMR per year into the further development of the privacy protocol if we also push users (via suggestion on getmonero.org and the reddit sidebar) into wallets that give users no choice but to give up their transaction data to propriatary black box data collection.   It makes no sense.

2. Considering the substantial amount of server resources required to monitor thousands+ if not hundreds of thousands of wallets in a light wallet server, doesn't it seem a little strange that a wallet company running these servers doesn't make it easy for users to use their own personal server resources to reduce compute cost?  Could it be that the in-app exchange offered by the Light Wallets isn't the only funding mechanism for their companies, and that they may be following the footsteps of countless other companies that sell user data?

3.  If  CakeWallet @SamsungGalaxyPlayer only allowed users to use Cake's Monero node, there would be a mass uprising in the Monero community.  Why does a wallet like Edge get a pass in this regard?  

Regarding phrasing:

Let me mull this one a bit.  I agree that short is always best, but it may be good to reach out to MRL or Monero-Dev to get an full and accurate understanding of the risks associated with these wallets in conjunction with a hosted server; it's a much worse situation than most people understand.

**Comment by SamsungGalaxyPlayer** [comment]
I'm for removing lightweight wallets that don't allow using a custom server. Maybe I'll change my mind with Seraphis/JAMTIS, but not now. The privacy leak is significant, and should not be recommended.

**Comment by plowsof** [comment]
MajesticBank noted on IRC that the edge app is the only one to have trackers https://reports.exodus-privacy.eu.org/en/reports/co.edgesecure.app/latest/ (if this would play into any decision making)

**Comment by jermanuts** [comment]
As I said at https://github.com/monero-project/monero-site/pull/2143#issuecomment-2002555434 you are putting a lot of trust in these Web-based wallets to not serve you a backdoored client from the server by dynamically serving JavaScript code to the browser to handle cryptography, creating wallets (stealing seeds) and when using an existing wallet etc.

They are much of security risk to recommend.

**Comment by detherminal** [comment]
changing the section's name might also be a good choice. feather is neither mobile nor light but it is still there.

**Comment by SyntheticBird45** [comment]
What about "Roaming" instead of "Lightweight". This is the only use case fwiw

**Comment by Dolorozo** [comment]
By the way, I found this post https://github.com/monero-project/monero-site/issues/2412
All criteria are met

**Comment by Dolorozo** [comment]
Hi @plowsof, @jermanuts, @erciccione ,
Sorry for pinging, but I think your input is important here. 

We’d love it if you could take a look at our request and share your thoughts. Would it be okay for us to open a **new PR** that reflects everything discussed so far? We have already covered all the pros and cons in the PR and in the comments.

As an extra note, our wallet has been reviewed and listed in several Monero directories (like Observer, Monerica). We know this isn’t a deciding factor, but we hope it shows our commitment. We’ve been supporting Monero for 4 years and have no plans to stop.

**Comment by nahuhh** [comment]
@Dolorozo do u still charge fees

**Comment by Dolorozo** [comment]
Hi again!
we honestly don’t understand why you keep saying this without checking the code. I ask you to spend a bit of time reviewing it or involve someone who **understands**. Judging by the repeated questions you haven’t looked. It's really hard to imagine, but it is possible to scan a tx without node. 

About Matrix reports. Over the last year there was only one mention about us. Yes, that’s more than 0, but not "numerous reports.” Why misinform? What’s the point? We do have support guys, I’ll ask them to respond faster, but we can’t be quicker than toxic comments no matter how hard we try :-)

Changelly - okay, we checked. We never recommend it manually (there’s no such functionality), the swap partner is chosen automatically by best rate. We will discuss this with the team asap.

Either we can open another PR to get proper attention from the developers. Otherwise it feels like off-top and no one will actually look at the code.
